extensions/interactions/FractionInput/directives/FractionInputValidationService.js

```diff
@@ -81,6 +81,19 @@ oppia.factory('FractionInputValidationService', [
             upperBoundConditionIsSatisfied;
         };
 
+        var shouldCheckRangeCriteria = function(rule1, rule2) {
```

Comment:
Hard to tell what the rule1, rule2 order is. Maybe call it earlierRule and laterRule?

Reply:
done